hw_3/hw_3.2.js

```diff
@@ -6,25 +6,23 @@ Task2*. Решить квадратные уравнения. Переменны
 */
 
 // quadraticEquation1 -> x2 - 6x + 9 = 0; 
-let x1;
-let x2;
 
-let a1 = 1;
-let b1 = -6;
-let c1 = 9;
+const a1 = 1;
+const b1 = -6;
+const c1 = 9;
 
-let discriminant1 = b1 ** 2 - 4 * a1 * c1;
+const discriminant1 = b1 ** 2 - 4 * a1 * c1;
+
+const x1 = (-b1 + Math.sqrt(discriminant1)) / (2 * a1);
+const x2 = (-b1 - Math.sqrt(discriminant1)) / (2 * a1);
```

Comment:
Looks like it's possible to improve the naming for this variables.
Provided formula is used to find the roots of a quadratic equation.
As an options: `root` or `quadraticRoot`

Reply:
thanks for your advices